src/main/java/utils/TabBar.java

```diff
@@ -1,12 +1,21 @@
-package pages;
+package utils;
```

Comment:
I think this is not a util but a component. A util class or method is nothing but a helper, but this TabBar class is a visual and functional component. I think you should move this to a package named `components` inside `pages`.
What do you think @WoloxAmquinte @gaboxxz ? We can discuss this later.

Reply:
I think it is the best, it is something visual and it is part of the pages, I will create the component package there